internal/oci/annotations_test.go

```diff
@@ -51,6 +54,8 @@ func TestProccessAnnotations_Expansion(t *testing.T) {
 	}
 
 	for _, tt := range tests {
+		tt := tt // TODO (go1.22) remove this since loop aliasing is avoided (https://go.dev/ref/spec#Go_1.22)
+
```

Comment:
Not sure I understand why this was done?

Reply:
opps, thats not necessary
removing